kore/src/Kore/Equation/Simplification.hs

```diff
@@ -80,18 +74,19 @@ simplifyEquation
     :: MonadSimplify simplifier
     => Equation RewritingVariableName
     -> simplifier (MultiAnd (Equation RewritingVariableName))
-simplifyEquation equation@(Equation _ _ _ _ _ _ _) =
```

Comment:
Why did we get rid of this? This was used to alert us with an error when the definition of `Equation` changed for whatever reason.